### PR TITLE
[OPENY-178] Include modules in package

### PR DESCRIPTION
**Pull request by anprok**

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1101 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1101 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1101 |
| Installation process | http://install.openy.cibox.tools/build1101/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1101/

File: modules/custom/openy_development/openy_development.info.yml [hunk 1]
```diff
@@ -11,3 +11,4 @@ dependencies:
   - domain
   - domain_theme_switch
 version: 8.x-1.0
+# @todo Module machine name should be changed to openy_ci after upgrade all sites to OpenY 2.0
```
Comment (ddrozdik) [inline modules/custom/openy_development/openy_development.info.yml:15]:
```suggestion
# @todo Module machine name should be changed to openy_ci after switching CI to OpenY 2.0
```

**Review by ddrozdik — CHANGES_REQUESTED** [review]

File: openy.packages.yml [hunk 2]
```diff
@@ -166,6 +166,9 @@ locations:
     - openy_prgf_amenities
     - openy_prgf_loc_finder
     - openy_prgf_location_by_amenities
+    - openy_field_hours
```
Comment (ddrozdik) [inline openy.packages.yml:157]: @anprok please check what is the difference between all 3 fields `openy_field_hours`,  `openy_field_custom_hours` and `openy_field_holiday_hours `. 
I think they will conflict with each other if enable all of them at the same time. I guess the original idea was to replace default fields with these two extended modules. If so, let's remove them from the list of package and create a new package for each of them.

**Review by ddrozdik — CHANGES_REQUESTED** [review]

File: openy.packages.yml (hunk 2 above)
Comment (podarok) [reply openy.packages.yml:157]: These are different fields, @ddrozdik

**Review by podarok — APPROVED** [review]

**Review by ddrozdik — APPROVED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1127 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1127 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1127 |
| Installation process | http://install.openy.cibox.tools/build1127/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1127/

**Pull request merged**